dallinger/recruiters.py

```diff
@@ -727,6 +736,60 @@ def get_qualifications(self, hit_id, sandbox):
             "peripheral_requirements": details["peripheral_requirements"],
         }
 
+    def screen_out(self, participant: Participant):
+        """Screen-out a participant.
+
+        Parameters:
+            participant: Participant
+                The participant to screen-out.
+        """
+        if self.screen_out_allowed(participant):
+            try:
+                return self.prolificservice.screen_out(
+                    study_id=self.current_study_id,
+                    submission_id=participant.assignment_id,
+                    bonus_per_submission=participant.base_pay + participant.bonus,
+                    increase_places=self.config.get("auto_recruit"),
+                )
+            except ProlificServiceException as ex:
+                logger.exception(str(ex))
```

Comment:
I don't think we should catch errors in this way, it means that logic errors don't get caught properly.

Reply:
This way of exception handling is partly used throughout `recruiters.py`, e.g. in `reward_bonus` and `approve_hit`. But yes, you are right, it is better to not catch these errors and only log them here. Removed the `try/except` therefore. Other methods like `recruit` also don't catch errors. So it looks somehow inconsistent how errors are treated.

Reply:
OK good!